### PR TITLE
CLI-1: Create initial folder structure for the project

### PR DESCRIPTION
**Pull request by RomanDrebotiy**
[CLI-1](https://github.com/RomanDrebotiy/swarm-mr-client/issues/1)
Created general code structure according to the ticket.

// @Gumbew @mranch

### CODE REVIEW

File: config/json/client_config.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+	"max_file_chunk_size_in_bytes": 4096,
+	"max_row_length": 123,
+	"map_reduce": {
+		"field_delimiter": "|",
+		"map_key_delimiter": "^",
+		"mapper_code_max_length": 12000,
+		"reducer_code_max_length": 12000,
```
Comment (RomanDrebotiy) [inline config/json/client_config.json]: Provided config values are considered as a default. We should also add command line interface options which can be used to override those defaults "on the fly".

File: client.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+def some_operation():
+    print('something...')
+
+if __name__ == '__main__':
+    some_operation()
```
Comment (RomanDrebotiy) [inline client.py]: So the basic rule is that we need to provide functions here for Python module interfeace and also add command line interface to call exactly the same functions.

File: config/json/client_config.json (hunk 1 above)
Comment (RomanDrebotiy) [reply config/json/client_config.json]: Also we can add other needed config here...

File: client.py (hunk 2 above)
Comment (RomanDrebotiy) [reply client.py]: Command line parameter parsing will be implemented using standard Python library modules.

File: client.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
+# TODO: public mdule interface goes here
```
Comment (mranch) [inline client.py]: "module" instead of "mdule"

Comment (mranch) [inline client.py]: "module" instead of "mdule"

Comment (RomanDrebotiy) [reply client.py]: Fixed.

**Pull request merged**